snippets/dockerfile.json

```diff
@@ -110,5 +110,14 @@
             "$0"
         ],
         "description": "ONBUILD"
+    },
+    "HEALTHCHECK": {
+        "prefix": "HEALTHCHECK",
+        "body": [
+            "HEALTHCHECK --interval=30s --timeout=30s --retries=3 CMD ${command}"
```

Comment:
>--interval=30s --timeout=30s --retries=3 [](start = 25, length = 40)

Do we need to provide these defaults? Does any of the above Keywords have any defaults?